x/incentive/types/msg.go

```diff
@@ -14,10 +14,11 @@ const (
 // Verify interface at compile time
 var _, _ sdk.Msg = &MsgWithdrawRewards{}, &MsgWithdrawValidatorCommission{}
 
-func NewMsgWithdrawRewards(delAddr sdk.AccAddress, denom string) *MsgWithdrawRewards {
+func NewMsgWithdrawRewards(delAddr sdk.AccAddress, denom string, withdrawType int64) *MsgWithdrawRewards {
 	return &MsgWithdrawRewards{
 		DelegatorAddress: delAddr.String(),
 		Denom:            denom,
+		WithdrawType:     withdrawType,
```

Comment:
Could add enum instead of int64 for withdrawType?

Reply:
We already have it in commitment module.

Reply:
fixed.